src/Tokenizer/Analyzer/CommentsAnalyzer.php

```diff
@@ -92,6 +92,11 @@ public function isBeforeStructuralElement(Tokens $tokens, $index)
         return false;
     }
 
+    public function isAnnotation(Token $token)
+    {
+        return 1 === Preg::match('#\/\*{2}\s+.*@\w+#m', $token->getContent());
```

Comment:
shouldn't this also do a type check? For example this string literal `$a = "/** @var int $foo */"` and HEREDOC/NOWDOC s etc., should all not be considered annotations I think?